ToyBox/classes/MonkeyPatchin/Vendor/BulkSellLogic.cs

```diff
@@ -310,7 +311,7 @@ internal class VendorVM_TryMassSalePatch {
                     vanillaMassSell = VendorHelper.IsAppropriateForMassSelling(item),
                     numberToSell = BulkSellLogic.canBulkSellCount(item)
                 })
-                .Where(item => item.item?.Owner != null && (item.vanillaMassSell || item.numberToSell > 0))
```

Comment:
This is all that was wrong. Looks like the owner is null most of the time. Maybe `Owner` gets set when a character actually equips the item?